internal/diff_str.ts

```diff
@@ -38,22 +36,9 @@ const EXT_LATIN_CHARS =
  */
 function tokenize(string: string, wordDiff = false): string[] {
   if (wordDiff) {
-    const tokens = string.split(WHITESPACE_SYMBOLS).filter((token) => token);
```

Comment:
the check for `tokens[i + 1]` would've always returned `true`, as we're filtering for truthy values.